config/docker-compose.yml

```diff
@@ -12,13 +11,11 @@ services:
       POSTGRES_USER: $USER
 
   elasticsearch:
-    container_name: dm-elasticsearch
     image: "elasticsearch:5.6"
     ports:
       - "9200:9200"
 
   nginx:
-    container_name: dm-nginx
```

Comment:
Does this mean our container names are going to be randomly suffixed, like `dmrunner-shrug-emoji`? I quite like having the shortcut of `dm-<service>` if I need to debug what's running and what isn't.

Reply:
Hmm, fair point.

The name scheme is fairly predictable, `<project name>_<service name>_<n>`, so for instance it would now be `dmrunner_nginx_1`.

Reply:
OK, that sounds fine 👍